Gifski/ConversionViewController.swift

```diff
@@ -16,12 +16,14 @@ final class ConversionViewController: NSViewController {
 	private lazy var timeRemainingEstimator = TimeRemainingEstimator(label: timeRemainingLabel)
 
 	private var conversion: Gifski.Conversion!
+	private var gifData: Data?	
 	private var progress: Progress?
 	private var isRunning = false
 
-	convenience init(conversion: Gifski.Conversion) {
+	convenience init(_ conversion: Gifski.Conversion, _ gifData: Data?) {
```

Comment:
Don't do unrelated changes.

```suggestion
	convenience init(conversion: Gifski.Conversion, _ gifData: Data?) {
```